client/app/serializers/workflow.js

```diff
@@ -0,0 +1,10 @@
+import DS from 'ember-data';
+
+
+export default DS.JSONSerializer.extend({
+
+    isSuccess: function(status, headers, payload) {
+        return status >= 200 && status < 300 || status === 304;
```

Comment:
If it's cached, it'll return 304 Not Modified, this lets ember know that its still successful.